docs/docsite/rst/os_guide/windows_winrm.rst

```diff
@@ -422,7 +422,7 @@ To configure Kerberos, in the section that starts with:
     [realms]
 
 Add the full domain name and the fully qualified domain names of the primary
-and secondary Active Directory domain controllers. It should look something
+and secondary Active Directory domain control nodes. It should look something
```

Comment:
@gotmax23 - another one for your eagle eyes. I think in this file, they may need to stay controllers as it's talking about Active Directory, not Ansible?